frontend/src/pages/AgencyProfileForm/AgencyProfileForm.js

```diff
@@ -268,6 +269,7 @@ class AgencyProfileForm extends Component {
   };
 
   render() {
+    if (isAuthenticated()) {
     const data = this.state;
```

Comment:
Indentation is not consistent -- down near the end of the if clause, it doesn't match up again. Consider inserting
```
if (!isAuthenticated()) {
  return <Redirect to="/login" />
}

return (
  ...
```
or doing the same thing as in other files, so we don't have to indent everything an additional level?

Reply:
By same thing as in other files, I mean
```
return ((!isAuthenticated()) ? <Redirect to='login' /> :
       (<div>
         <h2>Agency</h2>
         <CreateAgencyBtn />
       </div>)
```